broker_test.go

```diff
@@ -1,6 +1,53 @@
 package main
 
-import "testing"
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/vault/command"
+)
+
+const testToken = "root"
+
+func TestIntegration(t *testing.T) {
+	if err := setup(); err != nil {
+		t.Fatal(err)
+	}
+
+	t.Run("TestBroker_Services", TestBroker_Services)
+	t.Run("TestBroker_Provision", TestBroker_Provision)
+	t.Run("TestBroker_Deprovision", TestBroker_Deprovision)
+	t.Run("TestBroker_Bind", TestBroker_Bind)
+	t.Run("TestBroker_Unbind", TestBroker_Unbind)
+	t.Run("TestBroker_Update", TestBroker_Update)
+	t.Run("TestBroker_LastOperation", TestBroker_LastOperation)
+}
+
+func setup() error {
+
+	// Start Vault.
+	go command.Run([]string{"server", "-dev", "-dev-root-token-id="+testToken})
+
+	// Wait a few seconds for Vault to get started.
+	<- time.After(time.Second * 5)
```

Comment:
Is there no way to make this more deterministic, e.g. wait for the TCP port to be open?